inngest-core/build.gradle.kts

```diff
@@ -15,6 +15,8 @@ dependencies {
     implementation("com.fasterxml.jackson.core:jackson-databind:2.16.1")
     implementation("com.squareup.okhttp3:okhttp:4.12.0")
 
+    implementation("io.ktor:ktor-server-core:2.3.5")
```

Comment:
hm, I know you were looking at optional dependencies but I think it's a bit clearer if the framework specific adapters are wrappers outside of a generic `inngest-core`. Not sure what's more common in Kotlin/Java ecosystem though